marker_color_ranges: {
# red: {
#   r_min: 0.646789,
#   r_max: 0.876471,
#   g_min: 0.022222,
#   g_max: 0.110092,
#   r_avg: 0.745769,
#   g_avg: 0.056176
# },
# orange: {
#   r_min: 0.512545,
#   r_max: 0.623318,
#   g_min: 0.299652,
#   g_max: 0.379447,
#   r_avg: 0.568241,
#   g_avg: 0.332988
#  }
# pink: {
#   r_min: 0.379015,
#   r_max: 0.523711,
#   g_min: 0.181443,
#   g_max: 0.271949,
#   r_avg: 0.486951,
#   g_avg: 0.214402
# }
  yellow: {
    r_min: 0.495968,
    r_max: 0.554622,
    g_min: 0.423387,
    g_max: 0.478599,
    a_min: 0.294118,
    a_max: 0.335948,
    r_avg: 0.529259,
    g_avg: 0.452167,
    a_avg: 0.309418
  },
  green: {
    r_min: 0.245,
    r_max: 0.417021,
    g_min: 0.387234,
    g_max: 0.505,
    a_min: 0.258824,
    a_max: 0.335948,
    r_avg: 0.315603,
    g_avg: 0.453838,
    a_avg: 0.287606
  },
  blue: {
    r_min: 0.0287356,
    r_max: 0.194118,
    g_min: 0.223529,
    g_max: 0.336957,
    a_min: 0.222222,
    a_max: 0.294118,
    r_avg: 0.111558,
    g_avg: 0.288916,
    a_avg: 0.250853
  }
}
